config/webhook/manifests.yaml

```diff
@@ -89,9 +89,9 @@ webhooks:
     service:
       name: webhook-service
       namespace: system
-      path: /mutate-redhatcop-redhat-io-v1alpha1-databasesecretengineconfig
+      path: /mutate-redhatcop-redhat-io-v1alpha1-gcpauthengineconfig
```

Comment:
is this correct, it seems you are replacing the databasesecretengineconfig, this file is automatically generated, just run `make manifests`